### PR TITLE
Thomaslauria throwable patch

### PR DESCRIPTION
**Pull request by thomaslauria**
In Zend_Exception the previous type hint was still exception.

### CODE REVIEW

**Comment by thomaslauria** [comment]
Ah I just wanted to create a pull request for commit 2373501 - nevermind the other changes are in production in our application since over a year. I think they make sense too and could be merged after review.

**Comment by Shardj** [comment]
The throwable change is fine by the way, as are the rest of your changes. It's just the php unit stuff that's problematic, it's not the first time it's been suggested that we change the file.

File: library/Zend/Test/PHPUnit/ControllerTestCase.php [hunk 1]
```diff
@@ -37,14 +37,14 @@
...
  *
- * @uses       PHPUnit_Framework_TestCase
+ * @uses       PHPUnit\Framework\TestCase
  * @category   Zend
  * @package    Zend_Test
  * @subpackage PHPUnit
  * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
  * @license    http://framework.zend.com/license/new-bsd     New BSD License
  */
-abstract class Zend_Test_PHPUnit_ControllerTestCase extends PHPUnit_Framework_TestCase
```
[the remark this answers is not in the document]
Comment (thomaslauria) [reply library/Zend/Test/PHPUnit/ControllerTestCase.php:47]: I can not remember why we exactly did that change, I will have a look to this these days and will update the pull request then.

File: library/Zend/Test/PHPUnit/ControllerTestCase.php [hunk 2]
```diff
@@ -120,7 +120,7 @@ public function __get($name)
      *
      * Calls {@link bootstrap()} by default
      */
-    protected function setUp()
```
[the remark this answers is not in the document]
Comment (thomaslauria) [reply library/Zend/Test/PHPUnit/ControllerTestCase.php:123]: same here

**Comment by thomaslauria** [comment]
I removed the faulty PHPUnit 8 changes, since I could not discover any need in our project anymore.

**Review by Shardj — APPROVED** [review]

**Comment by Shardj** [comment]
Thanks for your help

**Pull request merged**